share/shutter/resources/modules/Shutter/App/OldCommon.pm

```diff
@@ -0,0 +1,447 @@
+###################################################
```

Comment:
Is this still needed?

Reply:
Just in case. It can be removed in the future

Reply:
If it needs to be restored, we have git history

Reply:
done in 12d391c4